ios/VizbeeNativeManager.m

```diff
@@ -203,6 +203,32 @@ -(void)sendEvent:(NSString*) name
     [[Vizbee getSessionManager] disconnectSession];
 }
 
+//----------------
+#pragma mark - Signin APIs
+//----------------
+
+RCT_EXPORT_METHOD(onGetSignInInfo:(NSDictionary*) signInInfo) {
+    RCTLogInfo(@"Invoking onGetSignInInfo");
+
+    VZBSessionManager* sessionManager = [Vizbee getSessionManager];
+    if (nil == sessionManager) {
+        RCTLogInfo(@"No session manager, session manager is nil");
+        return nil;
+    }
+
+    VZBSession* currentSession = [sessionManager getCurrentSession];
+    if (nil == currentSession) {
+        RCTLogInfo(@"No current session, current session is nil");
+        return nil;
+    }
+
+    NSMutableDictionary* authInfo = [NSMutableDictionary new];
+    [authInfo setValue:signInInfo forKey:@"authInfo"];
+
+    RCTLogInfo(@"Invoking sendEventWithName with authInfo = %@", authInfo);
+    [currentSession sendEventWithName:@"tv.vizbee.homesign.signin" andData:authInfo];
```

Comment:
Let's use contacts instead of magic numbers or string

Reply:
All magic strings are moved to VizbeeConstants file